.github/actions/job_init/action.yml

```diff
@@ -67,7 +67,6 @@ runs:
             echo "[INFO] The directory '${dependency_path}' doesn't exist. ${dependency_name} won't be removed."
           fi
         done
-        android_sdk_path="/usr/local/lib/android"
```

Comment:
Just wonder what was the reason to have it here?

Reply:
That was overlooked in the [FLINK-34360 PR](https://github.com/apache/flink/pull/24277/files#diff-47faf10cc8ebaecf32395ca48549315d271292595e4150968512e4df1fdb1721R70)